l10n_it_partially_deductible_vat/__openerp__.py

```diff
@@ -24,27 +24,19 @@
 ##############################################################################
 
 {
-    "name" : "Italy - Partially Deductible VAT",
-    "version" : "0.1",
-    "depends" : ['account'],
-    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
+    "name": "Italy - Partially Deductible VAT",
+    "version": "0.1",
+    "depends": ['account'],
+    "author": "OpenERP Italian Community,Odoo Community Association (OCA)",
     "description": """
```

Comment:
You should directly remove the ```description``` key from the dictionary, otherwise ```'\n       '``` willl be displayed